test/alternator/test_manual_requests.py

```
@@ -643,3 +648,61 @@ def test_write_malformed_value(dynamodb, test_table_s, op):
+    if url.scheme == 'https':
+        conn = http.client.HTTPSConnection(url.hostname, url.port, context=client_ssl_context(dynamodb), timeout=60)
+    else:
+        conn = http.client.HTTPConnection(url.hostname, url.port, timeout=60)
+
+    def do_request():
+        req = get_signed_request(dynamodb, 'DescribeTable', '{"TableName": "no_such_table"}')
+        conn.request('POST', '/', body=req.body, headers=req.headers)
+        response = conn.getresponse()
+        return response.status, response.read().decode('utf-8')
+
+    def wait_for(pred):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `wait_for`

(ANN202)